src/kOS.Safe/Exceptions/KOSArgumentMismatchException.cs

```diff
@@ -28,10 +27,19 @@ public override string HelpURL
         /// <param name="actual">number of actual arguments</param>
         /// <param name="message">optional message</param>
         public KOSArgumentMismatchException(int expected, int actual, string message = "" ) :
+            base( BuildTerseMessage(new[] {expected},actual) + message )
```

Comment:
Isn't it better to call another constructor of KOSArgumentMismatchException here?